readme.txt

```diff
@@ -56,4 +56,28 @@ To report a security issue, please visit the [WordPress HackerOne](https://hacke
 
 == Changelog ==
 
-See the [release history](https://github.com/wordpress/two-factor/releases).
+= 0.13.0 - 2025-04-02 =

```

Comment:
What's the expectation around this changelog and the `CHANGELOG.md`? Do we keep them in sync manually?

Reply:
Sounds good. We can probably implement some release magic that extracts the top items from the CHANGELOG file and places them here. That way we only need to update in a single location.